.github/workflows/smoke-test.yml

```diff
@@ -295,6 +295,8 @@ jobs:
         run: |
           shopt -s globstar
           pytest --nbmake source/**/*.ipynb -n=auto
+          # Messes with envvars, so can't be run in parallel!
+          FORCE_TEST_REPLACE_BINARY_PATH=1 pytest -k 'test_replace_binary_path'
```

Comment:
This is in the `python-notebooks` block: Should be under `python-test`? Unless it's here to avoid the test matrix?

Reply:
... but still, I think this should have located and run the test: Something else is wrong.

Reply:
Indeed something was wrong. Sorry for the iterations on this one.